examples/notebooks/use-cases/BBP KG Forge retrieval.ipynb

```diff
@@ -0,0 +1,167 @@
+      "\n",
+      "Not found\n",
+      "______________________\n",
+      "Execution 8 id same bucket Cross bucket True Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 9 self same bucket Cross bucket True Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 10 id other bucket Cross bucket True Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 11 self other bucket Cross bucket True Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 12 id same bucket Cross bucket False Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 13 self same bucket Cross bucket False Retrieve source False\n",
+      "rev 3\n",
+      "______________________\n",
+      "Execution 14 id other bucket Cross bucket False Retrieve source False\n",
+      "<action> catch_http_error\n",
+      "<error> RetrievalError: 404 Client Error: Not Found for url: https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0038048\n",
+      "\n",
+      "Not found\n",
+      "______________________\n",
+      "Execution 15 self other bucket Cross bucket False Retrieve source False\n",
+      "<action> retrieve\n",
+      "<error> RetrievalError: Provided resource identifier https://bbp.epfl.ch/nexus/v1/resources/neurosciencegraph/datamodels/_/http:%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0038048 is not inside the current bucket, use cross_bucket=True to be able to retrieve it\n",
+      "\n",
+      "Not found\n",
+      "______________________\n"
+     ]
+    }
+   ],
+   "source": [
+    "id_ = \"https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/20fbc97a-fb26-43ff-8093-9136aab25dff\"\n",
+    "self_ = \"https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/https:%2F%2Fbbp.epfl.ch%2Fnexus%2Fv1%2Fresources%2Fdke%2Fkgforge%2F_%2F20fbc97a-fb26-43ff-8093-9136aab25dff\"\n",
+    "\n",
+    "id_other_bucket = \"http://purl.obolibrary.org/obo/GO_0038048\"\n",
+    "self_other_bucket = \"https://bbp.epfl.ch/nexus/v1/resources/neurosciencegraph/datamodels/_/http:%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0038048\"\n",
+    "\n",
+    "all_ret = [\n",
+    "    (id_, \"id\", \"same bucket\"), \n",
+    "    (self_, \"self\", \"same bucket\"), \n",
+    "    (id_other_bucket, \"id\", \"other bucket\"),\n",
+    "    (self_other_bucket, \"self\", \"other bucket\"),\n",
+    "]\n",
+    "\n",
+    "i = 0\n",
+    "\n",
+    "for rs in [True, False]:\n",
+    "    for cb in [True, False]:\n",
+    "        for (el, type_, loc) in all_ret:\n",
+    "\n",
+    "            print(f\"Execution {i}\", type_, loc, \"Cross bucket\", cb, \"Retrieve source\", rs)\n",
```

Comment:
the printings may benefit from a separator between each parameter
`print(f"Execution {i} - ", type_, loc, "- Cross bucket:", cb, "- Retrieve source:", rs)`

Reply:
`print(f"Execution {i}", " using: ", type_, "inside: ", loc, " - Cross bucket:", cb, " - Retrieve source:", rs)`

Reply:
Changed to your suggestion for the print statement